src/utilities/date/FormatDate.js

```diff
@@ -0,0 +1,130 @@
+import moment from 'moment';
+
+const TOTAL_MONTH = 12;
+
+export const format = date => {
+  return moment(date).format('MMMM YYYY');
+};
+
+/**
+ * Convert month to year and month .
+ *
+ * @param {integer} monthCount
+ * @returns {Object}
+ */
+export const getYearMonth = monthCount => {
+  const m = monthCount % TOTAL_MONTH;
+  const y = Math.floor(monthCount / TOTAL_MONTH);
+
+  return { year: y, month: m };
+};
+
+export const getExperienceFormat = experience => {
+  if (experience.type === 'Year') {
+    return { year: Number(experience.value), month: 0 };
+  } else {
+    const result = getYearMonth(experience.value);
+
+    return { ...result };
+  }
+};
+
+/**
+ * Gett difference between two dates in years and months .
+ *
+ * @param {Date} startDate Start date.
+ * @param {Date} endDate  End date.
+ * @returns {Object} Difference.
+ */
+export const getDifferenceInYearMonth = (startDate, endDate) => {
+  const a = moment(endDate);
```

Comment:
please change the variable name.
May be like `formattedEndDate`. Because later on the line I forgot what a and b was. :D

Reply:
done